mu-plugins/blocks/global-header-footer/postcss/header/header.pcss

```diff
@@ -90,6 +90,39 @@ html {
 	}
 }
 
+.global-header__alert-banner {
+	--wp-global-header--link-color: #3e58e1;
```

Comment:
Should this color be `#3858e9` (`--wp--preset--color--blueberry-1`)?

Reply:
Yeah, thanks, fixed in ac660e528fbc409c2b2b9c3abc859aeeba68c89e